dirtify.py

```diff
@@ -12,6 +12,8 @@
 
 # construct the argument parser
 parser = argparse.ArgumentParser(description = "Filter a clean image, output a dirty one")
+parser.add_argument("-t", "--tile", action="store_true",
```

Comment:
I would add the `--tile` feature in a separate commit, to avoid muddying up the refactor. I prefer to see a commit consist of a refactor, a fix, or a new feature, but not to mix them. Makes for easier review and later reflection.

Reply:
Good point. Noted for next time.